packages/expo-doctor/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "expo-doctor",
+  "version": "1.0.0",
+  "type": "module",
+  "preferGlobal": true,
+  "keywords": [
+    "expo",
+    "ios",
+    "expo-env-info",
+    "android"
+  ],
+  "description": "Check your Expo project for known issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo-cli.git",
+    "directory": "packages/expo-doctor"
+  },
+  "author": "Expo <support@expo.dev>",
+  "license": "MIT",
+  "bin": {
+    "expo-doctor": "./build/index.js"
+  },
+  "files": [
+    "build"
+  ],
+  "scripts": {
+    "prepare": "yarn run clean && yarn run build:prod",
+    "lint": "eslint .",
+    "watch": "yarn run build -w",
+    "test": "jest",
+    "build": "ncc build ./src/index.ts -o build/",
+    "build:prod": "ncc build ./src/index.ts -o build/ --minify --no-cache --no-source-map-register",
+    "clean": "rimraf ./build/"
+  },
+  "engines": {
+    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Comment:
We dropped support for Node 12 in our tooling afaik (we also dropped Node 12 in our test matrix)